pymeasure/instruments/hp/hp856Xx.py

```diff
@@ -1027,12 +1031,31 @@ def set_continuous_sweep(self):
             if instr.done:
                 do_something()
 
-        """,
-        map_values=True,
-        values={True: "1"},
-        cast=str
+        """
+
     )
 
+    def check_done(self):
+        """
+        Get back (e.g. return) when all commands in a command string
+        entered before 'done' has been completed. Sending a :meth:`trigger_sweep` command
+        before 'done' ensures that the spectrum analyzer will complete a full sweep before
+        continuing on in a program.
+        Depending on the timeout a timeout error from the adapter will raise before the spectrum
+        analyzer can finish due to an extreme long sweep time.
```

Comment:
As it is not a property anymore, you are free to use other Docstrings, for example:
```suggestion
        Return when all commands in a command string
        entered before :meth:'check_done' has been completed. Sending a :meth:`trigger_sweep` command
        before 'check_done' ensures that the spectrum analyzer will complete a full sweep before
        continuing on in a program.
        Depending on the timeout a timeout error from the adapter will raise before the spectrum
        analyzer can finish due to an extreme long sweep time.
```